src/components/Pool/LoadingList/LoadingCard/index.tsx

```diff
@@ -1,140 +1,40 @@
+// @ts-nocheck
```

Comment:
I think we don't want to leave typecript disabled

Reply:
Thanks, fixed